pkg/scheduler/plugins/nodenumaresource/plugin.go

```diff
@@ -317,6 +322,46 @@ func (p *Plugin) Filter(ctx context.Context, cycleState *framework.CycleState, p
 	return nil
 }
 
+func (p *Plugin) filterAmplifiedCPUs(state *preFilterState, nodeInfo *framework.NodeInfo) *framework.Status {
+	quantity := state.requests[corev1.ResourceCPU]
+	podRequestMilliCPU := quantity.MilliValue()
+	if podRequestMilliCPU == 0 {
+		return nil
+	}
+
+	node := nodeInfo.Node()
+	ratios, err := extension.GetNodeResourceAmplificationRatios(node.Annotations)
```

Comment:
Nit: We can call the convenience method `cpuAmplificationRatio, err := extension.GetNodeResourceAmplificationRatio(node.Annotations, corev1.ResourceCPU)` here